ui-components/src/headers/HeadingGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { Heading } from "./Heading"
+import "./HeadingGroup.scss"
+
+export interface HeadingGroupProps {
+  /** A string or element to display in an `h2` tag (overridable via `headingPriority`) */
+  heading: React.ReactNode
+  /** A string or element to display in an `p` tag (using `aria-roledescription="subtitle"`) */
+  subheading: React.ReactNode
+  /**
+   * The heading level (1 through 6)
+   * @default 2
+   */
+  headingPriority?: number
+  /** Additional class name for the whole group */
+  className?: string
+  /** Additional class name for the heading */
```

Comment:
Why do we need one specific to each heading vs letting consumers pass in a root class name with overrides? I thought we may have been moving away from extra class names

Reply:
Good point, just having that "escape hatch" on the root is enough.